src/LogAnalysis/Classifier.hs

```diff
@@ -34,15 +34,15 @@ extractIssuesFromLogs :: (MonadCatch m) => [LogFile] -> Analysis -> m Analysis
 extractIssuesFromLogs logFiles analysis = do
 
     -- Will raise exception if we find an issue.
-    _                   <- mapM (checkFile . getLogFileContent) logFiles
+    _  <- checkFiles $ map getLogFileContent logFiles
```

Comment:
Is this the culprit then?

Reply:
@ksaric Yes!
Previously, we caught the error by checking each of the file.
But it seems like you cannot catch the error found on TSD-144 by doing that.
I guess the `logFiles` itself is throwing an error so when we try to perform `mapM`, the classifier instantly crashes because `logFiles` itself is an error.